jobs/openshift/cluster/integreatly/osd-cluster-integreatly-upgrade.yaml

```diff
@@ -0,0 +1,69 @@
+---
+- job:
+    name: osd-cluster-integreatly-upgrade
+    display-name: 'OSD Cluster Integreatly Upgrade'
+    project-type: pipeline
+    concurrent: false
+    parameters:
+      - string:
+          name: 'clusterName'
+          default: ''
+          description: '[REQUIRED] The name of the target cluster to upgrade Integreatly against'
+      - string:
+          name: 'upgradeGitBranch'
+          default: 'master'
+          description: '[REQUIRED] The name of the git branch to be used for the Integreatly installer'
+      - string:
+          name: 'routerShard'
+          default: ''
+          description: '[REQUIRED] Router shard of the targeted cluster'
+      - string:
+          name: 'openShiftToken'
+          default: ''
+          description: '[REQUIRED] Openshift token for cluster-admin user'
+      - string:
+          name: 'clientSecret'
+          description: '[REQUIRED] Desired client secret value for RHSSO'        
+    dsl: |
+        node('cirhos_rhel7') {
+          if (currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']){
+            currentBuild.description = "triggered by: " + currentBuild.getBuildCauses('hudson.model.Cause$UserIdCause')['userId']
+          }
+          currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+          stage('Upgrade Integreatly') {
+            wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+                ansibleTower(
+                  towerServer: 'Dev Tower',
+                  jobTemplate: 'Integreatly_Bootstrap_and_Upgrade_[OSD]',
+                  templateType: 'workflow',
+                  importTowerLogs: true,
+                  removeColor: false,
+                  verbose: true,
+                  extraVars: """---
+                    tower_environment: dev
+                    cluster_name: ${clusterName}
+                    integreatly_osd_update_branch: ${upgradeGitBranch}
+                    router_shard: ${routerShard}
+                    openshift_token: ${openShiftToken}"""
+                )
+            }  
+          }
+          
+          stage('Clone QE repository') {
+            dir('.') {
+              git branch: "master", url: "https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git"
+            }
+          }
+
+          stage('Update client secret in SSO') {
+            dir('test-suites/backend-testsuite') {
+              sh '''
+                export OPENSHIFT_TOKEN=${openShiftToken}
+                export CLUSTER_URL=https://console.${clusterName}.openshift.com
+                export SSO_URL=https://sso-openshift-sso.${routerShard}.${clusterName}.openshiftapps.com
+                npm install
+                node ssoClientUpdate.js ${clientSecret}
```

Comment:
I don't think so. Pre-upgrade we should have clientSercret properly set everywhere. Upgrade itself should not touch the clientSecret at all (at least it has not been so yet). Thus I think all the stuff related to this can be removed (stage here, input param at the top)

Reply:
Removed.Thx